### PR TITLE
RJP-55 : Fix Changes given by Jenkins plugin reviewer

### PR DESCRIPTION
**Pull request by ayushk048-Ajmera**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/redpen/service/RedpenJenkinsCore.java [hunk 1]
```diff
@@ -37,8 +38,8 @@ public void doPerform(ParameterModel parameter) {
 
         if (!StringUtils.isBlank(jwtToken)) {
             try {
-                List<String> allFiles = addAttachment(parameter, jwtToken);
-                addComment(parameter, jwtToken, allFiles);
+                List<String> allUploadedFileNames = addAttachment(parameter, jwtToken);
```
Comment (abhijitajmera) [inline src/main/java/org/jenkinsci/plugins/redpen/service/RedpenJenkinsCore.java:41]: If addAttachment handles multiple files, it's name should be addAttachments.

File: src/main/java/org/jenkinsci/plugins/redpen/secrets/SecretRetriever.java [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
+    public String getSecretFor(final String secretId, Run run) {
+        StringCredentials credentialById = CredentialsProvider.findCredentialById(secretId, StringCredentials.class, run);
 
-        final List<StringCredentials> credentials = getCredential();
-        final CredentialsMatcher matcher = CredentialsMatchers.withId(secretId);
-
-        return Optional.ofNullable(CredentialsMatchers.firstOrNull(credentials, matcher))
-                .flatMap(credential -> Optional.of(credential.getSecret()))
-                .flatMap(secret -> Optional.of(secret.getPlainText()));
+        if(credentialById == null) {
```
Comment (heenachauhan18) [inline src/main/java/org/jenkinsci/plugins/redpen/secrets/SecretRetriever.java:22]: Please format the code.

File: src/main/java/org/jenkinsci/plugins/redpen/secrets/SecretRetriever.java [hunk 3]
```diff
@@ -1,30 +1,29 @@
...
-import java.util.Optional;
 
 public class SecretRetriever {
     /**
      * Get Jenkins secret by secretId
      * @param secretId SecretId specified in Jenkins credential Manager
      * @return an Optional String if secret exist
      */
-    public Optional<String> getSecretFor(final String secretId) {
+    public String getSecretFor(final String secretId, Run run) {
```
Comment (heenachauhan18) [inline src/main/java/org/jenkinsci/plugins/redpen/secrets/SecretRetriever.java:19]: Please rename the method name a meaningful name.

File: src/main/java/org/jenkinsci/plugins/redpen/secrets/SecretRetriever.java (hunk 2 above)
Comment (heenachauhan18) [reply src/main/java/org/jenkinsci/plugins/redpen/secrets/SecretRetriever.java:22]: credentialById != null then do next steps.

File: src/main/java/org/jenkinsci/plugins/redpen/service/RedpenJenkinsCore.java [hunk 4]
```diff
@@ -58,29 +59,30 @@ public void doPerform(ParameterModel parameter) {
     private List<String> addAttachment(ParameterModel parameter, String jwtToken) throws IOException {
         RedpenService redpenService = RedpenService.getRedpenInstance();
 
-        List<String> allFiles = new ArrayList<>();
+        List<String> uploadedFilenameList = new ArrayList<>();
```
Comment (heenachauhan18) [inline src/main/java/org/jenkinsci/plugins/redpen/service/RedpenJenkinsCore.java:62]:
```suggestion
        List<String> uploadedFileNames = new ArrayList<>();
```

File: src/main/java/org/jenkinsci/plugins/redpen/service/RedpenJenkinsCore.java [hunk 5]
```diff
@@ -37,8 +38,8 @@ public void doPerform(ParameterModel parameter) {
 
         if (!StringUtils.isBlank(jwtToken)) {
             try {
-                List<String> allFiles = addAttachment(parameter, jwtToken);
-                addComment(parameter, jwtToken, allFiles);
+                List<String> allUploadedFileNames = addAttachments(parameter, jwtToken);
```
Comment (heenachauhan18) [inline src/main/java/org/jenkinsci/plugins/redpen/service/RedpenJenkinsCore.java:41]: uploadedFileNames

**Review by heenachauhan18 — APPROVED** [review]

**Review by heenachauhan18 — APPROVED** [review]

**Review by anandcpadiya — APPROVED** [review]

**Review by anandcpadiya — APPROVED** [review]

**Pull request merged**